Task_11/index.html

```diff
@@ -0,0 +1,308 @@
+
+        <table class="box">
+          <tr>
+            <th>File Name</th>
+            <th>File Size</th>
+            <th>Number of words</th>
+            <th>Remove</th>
+          </tr>
+          <tr>
+            <td>business_operations.docx</td>
+            <td>78kb</td>
+            <td>842</td>
+            <td>
+              <button type="button" class="trash"></button>
+            </td>
+          </tr>
+          <tr>
+            <td>business_financial.docx</td>
+            <td>26kb</td>
+            <td>2326</td>
+            <td>
+              <button type="button" class="trash"></button>
+            </td>
+          </tr>
+        </table>
+
+      </fieldset>
+
+      <fieldset>
+
+        <h2>Choose your languages pairings</h2>
+
+        <div class="selectWrapper">
+
+          <p>FROM</p>
+
+          <select name="initialLanguage" class="box">
+            <option value="EN">English</option>
+            <option value="FR">French</option>
+            <option value="UK" selected>Ukrainian</option>
+            <option value="FR">Italian</option>
+          </select>
+
+        </div>
+
+        <div class="selectWrapper">
+
+          <p>TO</p>
+
+          <select name="resultLanguage_1" class="box">
+            <option value="EN">English</option>
+            <option value="FR" selected>French</option>
+            <option value="UK">Ukrainian</option>
+            <option value="FR">Italian</option>
+          </select>
+            
+          <select name="resultLanguage_2" class="box">
+            <option value="EN" selected>English</option>
+            <option value="FR">French</option>
+            <option value="UK">Ukrainian</option>
+            <option value="FR">Italian</option>
+          </select>
+
+          <button type="button" class="trash"></button>
+
+          <button type="button" class="box">Add</button>
+
+        </div>
+
+      </fieldset>
+      
+      <fieldset class="tone">
+
+        <h2>Select your desired tone</h2>
+        
+        <div class="choicesWrapper">
+
+          <input hidden type="radio" id="casual" name="desiredTone" value="casual">
+          <label for="casual" class="box">Friendly / Casual</label>
+          
+          <input hidden type="radio" id="pro" name="desiredTone" value="pro" checked>
+          <label for="pro" class="box">Corporate / Pro</label>
+    
+          <input hidden type="radio" id="formal" name="desiredTone" value="formal">
+          <label for="formal" class="box">Media / Formal</label>
+
+        </div>
+
+      </fieldset>
+
+      <fieldset class="urgency">
+
+        <h2>Tell us how urgent this job is</h2>
+        
+        <div class="choicesWrapper">
+
+          <input type="range" name="urgent" min="-10" max="10" step="5" value="0">
+
+          <ul>
+            <li>I got time</li>
+            <li><strong>Average</strong></li>
+            <li>Yesterday</li>
+          </ul>
+
+          <div class="box">
```

Comment:
Semantic element for such block is <aside>

Reply:
It is not so obvious for me as well.
I have to absolutely position this `<legend>` to the bottom (as it should be the first child of the `<fieldset>`). Is it okay?
Also I can't put the text "With an average timeline set up, our expert translator can take a reasonable amount of time perfecting tour translation" inside a `<p>` tag (`<p>` isn't allowed inside `<legend>`). Then what element should contain that text?
Thanks